scss/components/_activities-calendar.scss

```diff
@@ -0,0 +1,176 @@
+$activities-calendar-active-day-background: #cde1ed;
```

Comment:
`$activity-no-result-icon-color` already has the same color. I suggest to check with Jon to give this color a new generic name. Which then can be used by both these vars.

Reply:
As discussed `$activity-no-result-icon-color` was removed since it's not being used.